### PR TITLE
Change label ID strings

### PR DESCRIPTION
**Pull request by maxjakob**
and align them with Ecosia's.

### CODE REVIEW

**Comment by maxjakob** [comment]
still pending approval on the Ecosia side from another team, but this would be my suggestion for the IDs

File: core/core/domain.py [hunk 1]
```diff
@@ -7,151 +7,152 @@
...
+    FWF = "certificate:FAIR_WEAR_FOUNDATION"
+    G_IT = "certificate:FUJITSU_GREEN_IT"
+    GAEA = "certificate:GAEA_ECOLOGICAL_AGRICULTURE"
+    GCS = "certificate:GOOD_CASHMERE_STANDARD"
+    GEPA = "certificate:GEPA"
+    GGN = "certificate:GGN"
+    GK = "certificate:GREEN_BUTTON"
+    GOTS = "certificate:GLOBAL_ORGANIC_TEXTILE_STANDARD"
+    GOTS_ORGANIC = "certificate:GLOBAL_ORGANIC_TEXTILE_STANDARD_ORGANIC"
+    GOTS_MWOM = "certificate:GLOBAL_ORGANIC_TEXTILE_STANDARD_ORGANIC"
```
Comment (se-jaeger) [inline core/core/domain.py:89]: Since this is the only one, I guess it's a mistake?

Comment (maxjakob) [reply core/core/domain.py:89]: yeah I'm also not sure about these 3. @se-jaeger  which source did you use to create `GOTS`, `GOTS_ORGANIC` and `GOTS_MWOM`

Comment (se-jaeger) [reply core/core/domain.py:89]: `GOTS` is somehow outdated. It's fromm Siegelklarheit but it does not really exists without an "extension". It's more like the "label provider" and "organic"/"made with organic materials" are the two label grades. E.g.: https://utopia.de/siegel/gots-siegel-global-organic-textile-standard/ -> "GOTS-Kriterien" first paragraph.

Comment (maxjakob) [inline core/core/domain.py:89]: @se-jaeger I see. so how about having only 2 and both have the threshold in the ID?

```suggestion
    GOTS_ORGANIC = "certificate:GLOBAL_ORGANIC_TEXTILE_STANDARD_95"
    GOTS_MWOM = "certificate:GLOBAL_ORGANIC_TEXTILE_STANDARD_70"
```

Comment (maxjakob) [reply core/core/domain.py:89]:
```suggestion
    GOTS = "certificate:GLOBAL_ORGANIC_TEXTILE_STANDARD"
    GOTS_ORGANIC = "certificate:GLOBAL_ORGANIC_TEXTILE_STANDARD_ORGANIC"
```

this is another proposal. the first would be 70%+, second one 90%+. That's how I thought about it in the other file:
https://github.com/ecosia/core/blob/dfb3c5e1948f84dd5f5b6555a3c4ac7b6466c0fd/shopping/sustainability-labels.json#L1322-L1356

Comment (se-jaeger) [reply core/core/domain.py:89]: Here: https://github.com/ecosia/core/blob/dfb3c5e1948f84dd5f5b6555a3c4ac7b6466c0fd/shopping/sustainability-labels.json#L1349
The ID and name do not fit.

**Comment by se-jaeger** [comment]
Ups didn't recognize this is directly a commit 😅

**Comment by se-jaeger** [comment]
@BigDatalex is currently working on preparing everything to use these IDs. If we agree on these IDs, I would like to hand this over to him. 👍🏼

File: core/core/domain.py (hunk 1 above)
Comment (maxjakob) [reply core/core/domain.py]: @se-jaeger I think I see what you mean. Does this fix it? https://github.com/ecosia/core/pull/9241

File: core/core/domain.py [hunk 2]
```diff
@@ -7,151 +7,151 @@
...
+    FT_TP = "certificate:FAIRTRADE_TEXTILE_PRODUCTION"
+    FWF = "certificate:FAIR_WEAR_FOUNDATION"
+    G_IT = "certificate:FUJITSU_GREEN_IT"
+    GAEA = "certificate:GAEA_ECOLOGICAL_AGRICULTURE"
+    GCS = "certificate:GOOD_CASHMERE_STANDARD"
+    GEPA = "certificate:GEPA"
+    GGN = "certificate:GGN"
+    GK = "certificate:GREEN_BUTTON"
+    GOTS_ORGANIC = "certificate:GLOBAL_ORGANIC_TEXTILE_STANDARD_95"
+    GOTS_MWOM = "certificate:GLOBAL_ORGANIC_TEXTILE_STANDARD_70"
```
Comment (maxjakob) [inline core/core/domain.py:88]:
```suggestion
    GOTS_ORGANIC = "certificate:GLOBAL_ORGANIC_TEXTILE_STANDARD_ORGANIC"
    GOTS_MWOM = "certificate:GLOBAL_ORGANIC_TEXTILE_STANDARD"
    GOTS = "certificate:GLOBAL_ORGANIC_TEXTILE_STANDARD"
```
sorry for breaking stuff with the last suggestion. I think this is the way we should handle it (2 IDs, one for regular, one for organic)

**Comment by se-jaeger** [comment]
Almost done, see #39. Are you ready on your end? So if the branch is ready to merge, can we directly deploy it?

**Comment by sleepypioneer** [comment]
I will check but I assume so :)

**Comment by sleepypioneer** [comment]
@se-jaeger we are ready :rocket:

**Comment by se-jaeger** [comment]
@maxjakob and @sleepypioneer, @BigDatalex and I just merged the changes and deployed everything to prod. You will get the new IDs from from Monday. 👍🏼 

Could you ping us from time to time if anything important in the labels JSON file changed? - Would be very helpful 😄 

Closing this since its a subset of #39

**Comment by sleepypioneer** [comment]
Awesome thank you both! And absolutely we will let you know, we should probably think of some way to automate this if it is changing often (and not by us directly) but lets see :)

**Pull request closed without merging**